source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Raspberry Pi All-In-One Installer"
+date: 2016-05-12 01:39
+comments: true
+sharing: true
+footer: true
+---
+Easily deploy a complete Home-Assistant server, complete with Websocket MQTT and Z-Wave support out of the box using Fabric!
+
+**Usage:**
```

Comment:
It's easier to link to a section if the titles are "linkable", eg. `{% linkable_title Usage %}`.